website/src/app/blog/sans-io/readme.mdx

```diff
@@ -644,12 +644,16 @@ Many thanks to [@algesten](https://github.com/algesten) for providing feedback
 on drafts of this post.
 
 [^1]:
-    Technically, a thread-per-core runtime could allow non-`'static` `Future`s.
+    For more details on Firezone's tech stack, see
+    [this article](/kb/architecture/tech-stack) in our architecture docs.
 
 [^2]:
+    Technically, a thread-per-core runtime could allow non-`'static` `Future`s.
+
+[^3]:
     `boringtun` does call `Instant::now` internally and is thus unfortunately
     partly impure, see https://github.com/cloudflare/boringtun/issues/391.
 
-[^3]:
+[^4]:
```

Comment:
Something is not right here:

![image](https://github.com/firezone/firezone/assets/5486389/0d0a1b2b-14e4-4ce0-b0b7-5420888f4517)